cmd/kubeapps-apis/buf.gen.yaml

```diff
@@ -12,3 +12,14 @@ plugins:
     out: gen
     opt:
       - module=github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen
+  # Consider migrating to openapiv3 once ready
+  # https://github.com/grpc-ecosystem/grpc-gateway/pull/2147
+  - name: openapiv2
+    out: docs
+    strategy: all
+    opt:
+      - allow_merge=true
+      - merge_file_name=kubeapps-apis
+      - allow_repeated_fields_in_body=true
+      # - generate_unbound_methods=true # have to re-check it
+      # - json_names_for_fields=false # have to re-check it
```

Comment:
I have to see the impact of these options. I have screened the ones that other people are using in other github projects, but I have to re-check when playing around with the generate API docs